pepper-apis/studybuilder-cli/studies/osteo/patches/osteo-consent-assent-pdf-v3.conf

```diff
@@ -0,0 +1,132 @@
+{
+  "pdfs": [
+    {
+      "name": "osproject-consent-assent",
+      "filename": "osproject-consent-assent",
+      "displayName": "Osteo parental consent & assent pdf",
+      "versions": [
+        {
+          "tag": "v3",
+          "start": null,
+          "end": null,
+          "sources": [
+            {
+              "type": "PARTICIPANT"
+            },
+            {
+              "type": "ACTIVITY",
+              "activityCode": "CONSENT_ASSENT",
+              "versionTag": "v3"
+            }
+          ],
+          "files": [
+            {
+              "filepath": "pdfs/consent_parental_v3.pdf",
+              "type": "CUSTOM",
+              "substitutions": [
+                {
+                  "type": "BOOLEAN",
+                  "field": "blood_draw_yes",
+                  "activityCode": "PARENTAL_CONSENT",
```

Comment:
I believe this is incorrect. Activity is CONSENT_ASSENT and we are referring to PARENTAL_CONSENT

Reply:
Looks like even Questions stable Ids  in THE PR should be updated too. In all places they seem to be from PARENTAL_CONSENT